FreeRTOS/History.txt

```diff
@@ -1,5 +1,17 @@
 Documentation and download available at https://www.FreeRTOS.org/
 
+Changes between FreeRTOS V10.4.1 and FreeRTOS V202011.00 released November 9 2020

```

Comment:
This is the history file for the FreeRTOS-Kernel repo, rathe rather than the FreeRTOS repo.  The FreeRTOS repo has new libraries, such as CoreMQTT, which should be mentioned here (with links to documentation).

Reply:
Addressed feedback in most recent PR